example/spec/parallel/testAPI/scheduledRuleWithExecutionPrefixSpec.js

```diff
@@ -0,0 +1,102 @@
+  waitForTestExecutionStart,
+} = require('@cumulus/integration-tests');
+
+const rulesApi = require('@cumulus/api-client/rules');
+const { deleteExecution } = require('@cumulus/api-client/executions');
+
+const { randomId } = require('@cumulus/common/test-utils');
+
+const {
+  createTestSuffix,
+  createTimestampedTestId,
+  loadConfig,
+  timestampedName,
+} = require('../../helpers/testUtils');
+
+describe('When I create a scheduled rule with an executionNamePrefix via the Cumulus API', () => {
+  let config;
+  let execution;
+  let executionName;
+  let executionNamePrefix;
+  let scheduledRuleName;
+  let scheduledHelloWorldRule;
+  let testSuffix;
+  let executionArn;
+  let beforeAllFailed;
+
+  const collectionsDir = './data/collections/s3_MOD09GQ_006';
+
+  beforeAll(async () => {
+    try {
+      config = await loadConfig();
+      process.env.stackName = config.stackName;
+
+      executionNamePrefix = randomId('prefix');
+
+      const testId = createTimestampedTestId(config.stackName, 'Rule');
+      testSuffix = createTestSuffix(testId);
+      scheduledRuleName = timestampedName('SchedRuleWithExecutionPrefix');
+      scheduledHelloWorldRule = {
+        name: scheduledRuleName,
+        collection: { name: `MOD09GQ${testSuffix}`, version: '006' },
+        workflow: 'HelloWorldWorkflow',
+        rule: {
+          type: 'scheduled',
+          value: 'rate(2 minutes)',
+        },
+        meta: {
+          triggerRule: scheduledRuleName,
+        },
+        executionNamePrefix,
+      };
+
+      await addCollections(config.stackName, config.bucket, collectionsDir,
+        testSuffix, testId);
+      // Create a scheduled rule
+      console.log(`creating rule ${scheduledRuleName}`);
+      await rulesApi.postRule({
+        prefix: config.stackName,
+        rule: scheduledHelloWorldRule,
+      });
+
+      execution = await waitForTestExecutionStart({
+        workflowName: scheduledHelloWorldRule.workflow,
+        stackName: config.stackName,
+        bucket: config.bucket,
+        findExecutionFn: (taskInput, params) =>
+          taskInput.meta.triggerRule && (taskInput.meta.triggerRule === params.ruleName),
+        findExecutionFnParams: { ruleName: scheduledRuleName },
+        startTask: 'HelloWorld',
+      });
+
+      executionArn = execution.executionArn;
+      executionName = execution.executionArn.split(':').reverse()[0];
+    } catch (error) {
+      beforeAllFailed = true;
+      throw error;
+    }
+  });
+
+  afterAll(async () => {
+    console.log(`deleting rule ${scheduledRuleName}`);
+
+    await rulesApi.deleteRule({
+      prefix: config.stackName,
+      ruleName: scheduledRuleName,
+    });
+    await deleteExecution({ prefix: config.stackName, executionArn });
+
+    await cleanupCollections(config.stackName, config.bucket, collectionsDir,
+      testSuffix);
+  });
+
+  it('the triggered execution has the requested prefix', () => {
+    if (beforeAllFailed) fail('beforeAll() failed');
```

Comment:
When adding this boilerplate, it'd be nice to have a pattern where instead of just noting the before all fails, we instead throw the original error, or at least output it.    We've used something like this in a few other int tests recently: 

e.g. set beforeAllFailed (renamed to beforeAllError) *to* the error, then 

if (beforeAllFailed) fail(beforeAllFailed);

or some variation on it.